### DISCUSSION TITLE
Shell theme shown in the pdm.fming.dev video?

### DISCUSSION THREAD

**Discussion by mpsmith** [pdm-project/pdm#1488] [Q&A] [answered]
I like the shell `theme` in the first demo `asciinema` video on `pdm.fming.dev`.

Would you be so kind as to share if its custom or is available somewhere?

It looks a little like `Oh My Zsh` theme `agnoster`.

**Comment by frostming** [comment, marked as answer]
[Powerlevel 10k](https://github.com/romkatv/powerlevel10k)
